PopulateTools/gobierto
# Liquidify layout

Closes [#288](/PopulateTools/issues/issues/288)

## :v: What does this PR do?

This PR simplifies the layout of a site and allows it to be edited in the admin UI.

## :mag: How should this be manually tested?

You should be able to:

1. edit the site layout
2. check the custom layout is being rendered instead of the default

## :eyes: Screenshots

### Before this PR

🦆 

### After this PR

![screen shot 2018-04-02 at 16 44 51](https://user-images.githubusercontent.com/17616/38200577-357b67d2-3695-11e8-91d0-7b1ce9e0a117.png)
